modules/es-cluster-health-is-red.adoc

```diff
@@ -0,0 +1,153 @@
+
+. If it seems like the recovery has stalled, check if the `cluster.routing.allocation.enable` value is set to `none`, by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/settings?pretty
+----
+
+. If the `cluster.routing.allocation.enable` value is set to `none`, set it to `all`, by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/settings?pretty -X PUT -d '{"persistent": {"cluster.routing.allocation.enable":"all"}}'
+----
+
+. Check if any indices are still red by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/indices?v
+----
+
+. If any indices are still red, try to clear them by performing the following steps.
+
+.. Clear the cache by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_cache/clear?pretty
+----
+
+.. Increase the max allocation retries by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.allocation.max_retries":10}'
+----
+
+.. Delete all the scroll items by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_search/scroll/_all -X DELETE
+----
+
+.. Increase the timeout by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.unassigned.node_left.delayed_timeout":"10m"}'
+----
+
+. If the preceding steps do not clear the red indices, delete the indices individually.
+
+.. Identify the red index name by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/indices?v
+----
+
+.. Delete the red index by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_red_index_name> -X DELETE
+----
+
+. If there are no red indices and the cluster status is red, check for a continuous heavy processing load on a data node.
+
+.. Check if the Elasticsearch JVM Heap usage is high by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_nodes/stats?pretty
```

Comment:
```suggestion
$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query="_nodes/stats?pretty"
```

Reply:
Per https://redhat-internal.slack.com/archives/C02321AKYTT/p1696866637850469?thread_ts=1696524477.252959&cid=C02321AKYTT we will leave this as is